src/Renci.SshNet/SftpClient.cs

```diff
@@ -2123,26 +2358,26 @@ private void InternalUploadFile(Stream input, string path, Flags flags, SftpUplo
 
                 if (bytesRead > 0)
                 {
-                    var writtenBytes = offset + (ulong) bytesRead;
+                    var writtenBytes = offset + (ulong)bytesRead;
 
                     _sftpSession.RequestWrite(handle, offset, buffer, 0, bytesRead, null, s =>
+                    {
+                        if (s.StatusCode == StatusCodes.Ok)
```

Comment:
Please undo formatting changes.

Reply:
@vanillajonathan You forgot to thumb down this one. Thanks for the constructive feedback.